com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConverter.cs

```diff
@@ -19,7 +19,7 @@ internal abstract class RenderPipelineConverter
         /// <summary>
         /// A check if the converter is enabled or not. Can be used to do a check if prerequisites are met to have it enabled or disabled.
         /// </summary>
-        public virtual bool IsEnabled { get => m_IsEnabled; set => m_IsEnabled = value; }
+        public virtual bool IsEnabled => m_IsEnabled;
```

Comment:
Get rid of `m_IsEnabled`, return `true` instead

Reply:
Done